docs/src/modules/components/DemoToolbar.js

```diff
@@ -109,36 +108,30 @@ const Button = styled(MDButton)(({ theme }) => ({
   flexShrink: 0,
   borderRadius: 999,
   border: 'none',
-  paddingLeft: theme.spacing(1),
-  paddingRight: theme.spacing(0.5),
   fontSize: theme.typography.pxToRem(13),
   fontWeight: theme.typography.fontWeightMedium,
-  color: theme.palette.primary.main,
+  color: theme.palette.text.secondary,
   '& svg': {
-    fill: theme.palette.primary.main,
+    color: theme.palette.text.secondary,
   },
   [`:hover`]: {
-    backgroundColor: theme.vars
-      ? `rgba(${theme.vars.palette.primary.mainChannel} / calc(${theme.vars.palette.action.selectedOpacity} + ${theme.vars.palette.action.hoverOpacity}))`
-      : alpha(
-          theme.palette.primary.main,
-          theme.palette.action.selectedOpacity + theme.palette.action.hoverOpacity,
-        ),
+    backgroundColor: theme.palette.grey[50],
     '@media (hover: none)': {
-      backgroundColor: theme.vars
-        ? `rgba(${theme.vars.palette.primary.mainChannel} / ${theme.vars.palette.action.selectedOpacity})`
-        : alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity),
+      backgroundColor: theme.palette.grey[50],
```

Comment:
I search on the rest of the docs. It's missing the following comment before `'@media (hover: none)'`
```
// Reset on touch devices, it doesn't add specificity
```

According to [docs](https://drafts.csswg.org/mediaqueries-4/#hover) `@media (hover: none)` target device which main input does not support hover (mobile, tablette, ...)

I assume this has been added to avoid adding some color changes when the finger is hover the button

If you put two time the same color for both, we can simply remove the second one. Same for the dark mode

Reply:
Added the comment! Anything else you think I should do? I haven't removed the declaration entirely because it was there already before.

Reply:
But it was there with a different property

`:hover`: `theme.palette.action.selectedOpacity + theme.palette.action.hoverOpacity`
`@media (hover: none)`: `theme.palette.action.selectedOpacity`

Reply:
Uhm, I see. 🤔  I was trying to simplify that part a bit since the visual difference wasn't that big ⎯ does it break anything using the simpler color declaration there?

Reply:
The issue

https://github.com/mui/material-ui/assets/3165635/723870eb-2d31-4de5-b366-d03561a14fd7